python/ray/serve/constants.py

```diff
@@ -99,6 +101,9 @@
 # Handle metric push interval. (This interval will affect the cold start time period)
 HANDLE_METRIC_PUSH_INTERVAL_S = 10
 
+# Timeout for GCS internal KV service
+RAY_SERVE_KV_TIMEOUT = int(os.environ.get("RAY_SERVE_KV_TIMEOUT_S", "0")) or None
```

Comment:
Does this mean by default, Serve doesn't time out during KV operations?

Reply:
same question? Why is there a None?

Reply:
also can you suffix this constant with _S like others?

Reply:
also this can be a float right?

Reply:
@simon-mo @shrekris-anyscale maybe I didn't state it clearly. My plan is to put a non-None value in following PR since it involves more work. This is only for replacing GcsClient in serve. The WIP PR is [here](https://github.com/ray-project/ray/pull/25637).

@simon-mo  Good catch I'll update the int to float.

I'm not sure whether it's a good practice to add _S there because, for python, all the timeouts are measured by seconds in float type. But it's your call.

Reply:
_S is the convention ins Serve, update will be great!

Reply:
To clarify, you are saying that right now gcs client only supports timeout=None?

Reply:
No, GCS client supports timeout. I only mean, for this PR, serve is not **using** timeout. The timeout will be enabled in the next PR as mentioned.